keras_nlp/models/albert/albert_masked_lm_preprocessor.py

```diff
@@ -68,6 +68,15 @@ class AlbertMaskedLMPreprocessor(AlbertPreprocessor):
                     left-to-right manner and fills up the buckets until we run
                     out of budget. It supports an arbitrary number of segments.
 
+        Call arguments:
```

Comment:
remove indent (should align with Args)

Reply:
This comment has not been addressed, please fix it, thanks!